ext/libxml/libxml.c

```diff
@@ -1095,6 +1095,25 @@ PHP_FUNCTION(libxml_use_internal_errors)
 }
 /* }}} */
 
+static void php_libxml_create_error_object(zval *return_value, const xmlError *error)
+{
+	object_init_ex(return_value, libxmlerror_class_entry);
+	add_property_long(return_value, "level", error->level);
+	add_property_long(return_value, "code", error->code);
+	add_property_long(return_value, "column", error->int2);
+	if (error->message) {
+		add_property_string(return_value, "message", error->message);
+	} else {
+		add_property_stringl(return_value, "message", "", 0);
```

Comment:
Does it make sense to use the zend_string version and pass in the "global" empty string?